lua/onedark/util.lua

```diff
@@ -209,53 +184,53 @@ function util.terminal(colors)
 end
 
 function util.light_colors(colors)
-  if type(colors) == "string" then
-    return util.getColor(colors)
-  end
+  if type(colors) == "string" then return util.getColor(colors) end
   local ret = {}
-  for key, value in pairs(colors) do
-    ret[key] = util.light_colors(value)
-  end
+  for key, value in pairs(colors) do ret[key] = util.light_colors(value) end
   return ret
 end
 
 ---@param theme Theme
 function util.load(theme)
--- only needed to clear when not the default colorscheme
-  if vim.g.colors_name then
-    vim.cmd("hi clear")
-  end
-  -- if vim.fn.exists("syntax_on") then
-    -- vim.cmd("syntax reset")
-  -- end
+  vim.cmd("hi clear")
+  if vim.fn.exists("syntax_on") then vim.cmd("syntax reset") end
 
   vim.o.termguicolors = true
   vim.g.colors_name = "onedark"
   -- vim.api.nvim__set_hl_ns(ns)
   -- load base theme
   util.syntax(theme.base)
-  util.syntax(theme.plugins)
-  util.terminal(theme.colors)
-  util.autocmds(theme.config)
+
+  -- load syntax for plugins and terminal async
+  vim.defer_fn(function()
+    util.terminal(theme.colors)
+    util.syntax(theme.plugins)
+    util.autocmds(theme.config)
+  end, 0)
```

Comment:
This is awesome, I believe this is the fix for the bug! Do you know why this works? I assume it prevents a race condition with overriding the `lualine` inactive config?